install_config/registry/securing_and_exposing_registry.adoc

```diff
@@ -193,6 +193,27 @@ cf2616975b4a: Image successfully pushed
 Digest: sha256:3662dd821983bc4326bee12caec61367e7fb6f6a3ee547cbaff98f77403cab55
 ----
 
+. When using authentication, some versions of Docker also require you to trust the
+certificate at the OS level.
```

Comment:
Is "require you to trust" the correct working? Would it be more accurate to say "...require you to configure your cluster to trust..."?